dags/eam_workday_xmatters_integration.py

```diff
@@ -117,7 +117,9 @@ def create_jira_ticket(context):
     "owner": "jmoscon@mozilla.com",
     "emails": ["jmoscon@mozilla.com"],
     "start_date": datetime(2024, 1, 1),
-    "retries": 0,
+    "retries": 3,
+    # wait 5 min before retry
+    "retry_delay": datetime.timedelta(minutes=5),

```

Comment:
Above import is `datetime.datetime` 
```suggestion
    "retry_delay": timedelta(minutes=5),
```
and you can update import to:
`from datetime import datetime, timedelta`